core/src/com/unciv/logic/automation/ChooseBeliefsAutomation.kt

```diff
@@ -99,33 +100,53 @@ object ChooseBeliefsAutomation {
                     unique.stats.values.sum() *
                         if (city.isConnectedToCapital()) 2f
                         else 1f
-                UniqueType.StatPercentFromReligionFollowers -> 
-                    min(unique.params[0].toFloat() * city.population.population, unique.params[2].toFloat()) 
+                UniqueType.StatPercentFromReligionFollowers ->
+                    min(unique.params[0].toFloat() * city.population.population, unique.params[2].toFloat())
                 UniqueType.StatsPerCity ->
                     if (city.matchesFilter(unique.params[1]))
                         unique.stats.values.sum()
                     else 0f
                 else -> 0f
             }
         }
-        
+
         return score
     }
-    
+
     private fun beliefBonusForPlayer(civInfo: CivilizationInfo, belief: Belief): Float {
         var score = 0f
-        val amountOfEnhancedReligions = civInfo.religionManager.amountOfFoundableReligions()
+        val numberOfFoundedReligions = civInfo.gameInfo.civilizations.count {
+            it.religionManager.religion != null && it.religionManager.religionState >= ReligionState.Religion
+        }
+        val maxNumberOfReligions = numberOfFoundedReligions + civInfo.religionManager.remainingFoundableReligions()
+
+        // adjusts scores of certain beliefs as game evolves (adapted from Civ 5 DLL files on AI belief selection)
+        // enable differentiation of early vs late founding of religion and early vs late enhancement of religion
+        // this is mainly for mods which may shuffle enhancer and founder beliefs w.r.t. base UnCiv
+        var gameTimeScalingPercent = 50
+        when (civInfo.religionManager.religionState) {
+            ReligionState.FoundingReligion -> {
+                gameTimeScalingPercent = 100 - ((numberOfFoundedReligions * 100) / maxNumberOfReligions)
+            }
+            ReligionState.EnhancingReligion -> {
+                val amountOfEnhancedReligions = civInfo.gameInfo.civilizations.count {
+                    it.religionManager.religion != null && it.religionManager.religionState == ReligionState.EnhancedReligion
+                }
+                gameTimeScalingPercent = 100 - ((amountOfEnhancedReligions * 100) / maxNumberOfReligions)
+            }
+            else -> {} // pantheon shouldn't matter
+        }
         val goodEarlyModifier = when {
-            amountOfEnhancedReligions < 33 -> 1f
-            amountOfEnhancedReligions < 66 -> 2f
+            gameTimeScalingPercent < 33 -> 1f
+            gameTimeScalingPercent < 66 -> 2f
             else -> 4f
         }
         val goodLateModifier = when {
-            amountOfEnhancedReligions < 33 -> 2f
-            amountOfEnhancedReligions < 66 -> 1f
+            gameTimeScalingPercent < 33 -> 2f
+            gameTimeScalingPercent < 66 -> 1f
             else -> 1/2f
-        } 
-        
+        }
```

Comment:
Previously, `amountOfEnhancedReligions` just counted the number of foundable religions which breaks the modifier since the modifier assumed integer percentages (0-100). This meant that the AI always believed it was late game (`goodLateModifier = 2f`). However, due to the `goodEarlyModifier`/`goodLateModifier` being divided further in the function, these modifiers actually had the opposite effect than they were intended. This has been fixed below so the modifiers are multiplied.